changelog/std-algorithm-iteration-each-early-stopping.dd

```diff
@@ -0,0 +1,19 @@
+`std.algorithm.iteration.each` is now capable of early-stopping
+
+$(REF each, std,algorithm,iteration) is now capable of exiting early.
+When a `No.each` $(REF Flag, std,typecons) is passed to `each`, the iteration will be aborted early.
+For example:
```

Comment:
(Minor) I had to read this a couple times to get it. Perhaps better would be if the wording describes what the iteration function returns rather than what is passed to `each`. A slightly different example showing both yes and no cases might also do this. eg.
```
auto arr = [10, 20, 30];
arr.each!((n) { arr ~= n; return (n == 20) ? No.each : Yes.each; });
assert(arr == [10, 20, 30, 10, 20]);
```

Reply:
Good idea. I tried to rephrase it.